### DISCUSSION TITLE
running multiple models of the same data in the same directory

### DISCUSSION THREAD

**Discussion by jc-brandeis** [gelles-brandeis/tapqir#370] [Q&A] [answered]
Can we run different models of the same data in the same working directory?

**Comment by ordabayevy** [comment, marked as answer]
Yes. The output files for different models are saved with different names prefixed with the model name. For example, the parameter output files in matlab format will be saved as `cosmos_params.mat` and `cosmos+hmm_params.mat` files for `cosmos` and `cosmos+hmm` models respectively.
